CHANGES.md

```diff
@@ -2,6 +2,9 @@
 
 ### Added
 
+- Add a `dune-release config` subcommand to display and edit the global
```

Comment:
Wouldn't it be better to call it "local configuration" instead of "global configuration" ?

Reply:
Yeah I see why it can be confusing. By global I meant system wide or user config, as opposed to the per-project config we might add soon. Maybe we should use a different terminology here to avoid confusion.

What do you think?

Reply:
Oh right, I guess it makes sense then.